tool/deploy.sh

```diff
@@ -38,7 +38,7 @@ fi
 if [[ -z $LOCAL && -z "$FIREBASE_TOKEN" ]]; then
   _error "Cannot deploy, the FIREBASE_TOKEN environment variable isn't defined. Did you mean to use --local? Use --help for details."
 elif [[ -z $LOCAL ]]; then
-  _ARGS+=" --non-interactive --token $FIREBASE_TOKEN"
```

Comment:
There is no need to pass this, because firebase-tools picks up the environment variable when set. In fact, there are reasons not to pass it because the token can be leaked with e.g. a process listing.

Reply:
Good point!